docs/application/native/guides/internationalization/i18n.md

```diff
@@ -76,7 +76,7 @@ The main features of the i18n API include:
   You can [format plural messages](#plural_format), with PluralFormat.
 
 > **Note**  
-> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported since Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported since Tizen 4.0. The Immutable Index and the PluralFormat is supported since Tizen 5.0.
+> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported from Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported from Tizen 4.0. The Immutable Index and the PluralFormat is supported from Tizen 5.0.
```

Comment:
Please change as:

The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported since Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported since Tizen 4.0. The Immutable Index and the PluralFormat is supported from Tizen 5.0.

Reply:
@sabeena21 , I have a question for "since/from Tizen" In this sentence we have "since Tizen 3.0", "since Tizen 4.0" and "from Tizen 5.0"

Don't we need to use one word in above cases?